packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts

```diff
@@ -300,6 +302,25 @@ export default function setAlertText(alert: any, callback: Function) {
         );
         break;
 
+      // PublicCreateOrders
+      case PUBLICCREATEORDERS:
+        if (alert.params._prices.length === 1) {
+          alert.title = 'Order placed';
+          alert.details = 'Your order has been added';
+        } else {
+          alert.title = 'Orders placed';
+          alert.details = 'Your orders have been added';
+        }
+        dispatch(
+          loadMarketsInfoIfNotLoaded([marketId], () => {
+            const marketInfo = selectMarket(marketId);
+            if (marketInfo === null) return;
+            alert.description = marketInfo.description;
+            alert.toast = true;
```

Comment:
i think we need to put the toast: true part when the alert first gets added. or else on refresh the toast will show up again

Reply:
Yeah, true. I'll update that as well.